config.yaml

```diff
@@ -36,6 +36,10 @@ options:
       Comma separated list of IP address CIDR ranges that should be allowed for
       federation, identity servers, push servers, and for checking key validity
       for third-party invite events.
+  notif_from:
+    type: string
+    description: defines the "From" address to use when sending emails.
+      It must be set if email sending is enabled. Defaults to server_name.
```

Comment:
- Instead of "email sending" something like "SMTP integration"?
- I think we discussed that before the break: isn't this description mistaken?

From the docs, the default would be "Matrix", right?
https://matrix-org.github.io/synapse/latest/usage/configuration/config_documentation.html#email

Reply:
Changed to SMTP integration as it is clearer than "email sending". I did not add the "%(app)'s" info as app_name is not set in the charm (and as said, just defaults to Matrix)